src/Service/ActivityPubManager.php

```diff
@@ -844,6 +844,47 @@ public function extractMarkdownSummary(array $apObject): ?string
         }
     }
 
+    public function extractMarkdownContent(array $apObject)
+    {
+        if (isset($apObject['source']) && isset($apObject['source']['mediaType']) && isset($apObject['source']['content']) && ApObjectExtractor::MARKDOWN_TYPE === $apObject['source']['mediaType']) {
+            return $apObject['source']['content'];
+        } else {
+            $converter = new HtmlConverter(['strip_tags' => true]);
+
+            return stripslashes($converter->convert($apObject['content']));
+        }
+    }
+
+    public function isActivityPublic(array $payload): bool
```

Comment:
This method is no longer used, correct?

Reply:
See: https://github.com/MbinOrg/mbin/pull/830#discussion_r1659967075